### PR TITLE
ipconfig: add page and Chinese translation

### PR DESCRIPTION
**Pull request by hellojukay**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/ipconfig.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ipconfig
+
+> View and control IP configuration state.
+> More information: <https://ss64.com/osx/ipconfig.html>.
+
+- List all network interface:
```
Comment (kbdharun) [inline pages/osx/ipconfig.md:6]:
```suggestion
- List all network interfaces:
```

File: pages/osx/ipconfig.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`ipconfig getiflist`
+
+- Get interface address:
+
+`ipconfig getifaddr {{interface name}}`
```
Comment (kbdharun) [inline pages/osx/ipconfig.md:12]:
```suggestion
`ipconfig getifaddr {{interface_name}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/osx/ipconfig.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# ipconfig
+
+> View and control IP configuration state.
+> More information: <https://ss64.com/osx/ipconfig.html>.
+
+- List all network interfaces:
+
+`ipconfig getiflist`
+
+- Get interface address:
```
Comment (acuteenvy) [inline pages/osx/ipconfig.md:10]:
```suggestion
- Show the IP address of an interface:
```

File: pages.zh/osx/ipconfig.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# ipconfig
+
+> 查看和编辑网卡信息.
+> 更多信息: <https://ss64.com/osx/ipconfig.html>.
```
Comment (blueskyson) [inline pages.zh/osx/ipconfig.md:4]:
```suggestion
> 更多信息：<https://ss64.com/osx/ipconfig.html>.
```

File: pages.zh/osx/ipconfig.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+- 列出所有的网卡:
```
Comment (blueskyson) [inline pages.zh/osx/ipconfig.md:6]:
```suggestion
- 列出所有的网卡：
```

File: pages.zh/osx/ipconfig.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+`ipconfig getiflist`
+
+- 获取特定网卡的地址信息:
+
+`ipconfig getifaddr {{interface_name}}`
```
Comment (blueskyson) [inline pages.zh/osx/ipconfig.md:12]:
```suggestion
- 获取特定网卡的地址信息：

`ipconfig getifaddr {{接口名称}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by acuteenvy — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**